constants/routes.go

```diff
@@ -0,0 +1,19 @@
+package constants
+
+var WebRoutes = map[string]map[string]string{
+	"session": {
```

Comment:
Should it be plural?
```suggestion
	"sessions": {
```

Reply:
Yes, fixed in [020218e](https://github.com/carryall/go-google-scraper-challenge/pull/116/commits/020218e0c5c65eca931880158c4d5de4fb2c9828) along with the comments below 🙇🏼